src/pages/settings/Profile/ProfilePage.js

```diff
@@ -162,10 +160,6 @@ class ProfilePage extends Component {
      * Submit form to update personal details
      */
     updatePersonalDetails() {
-        if (!this.validateInputs()) {
-            return;
-        }
-
         // Check if the user has modified their avatar
         if ((this.props.myPersonalDetails.avatar !== this.state.avatar.uri) && this.state.isAvatarChanged) {
```

Comment:
The Form submit handler takes in `values` as a param and we should use them instead of local state values.